dwitter/feed/views.py

```diff
@@ -83,6 +83,7 @@ def feed(request, page_nr, sort):
                'header_title': 'Dwitter',
                'feed_type': 'all',
                'page_nr': page,
+               'on_last_page': last == dweet_count,
```

Comment:
Should be 'last >= dweet_count' as last is simply "page * dweets_per_page" so I'm guessing this will fail when the number of dweets are not a multiplum of 10.